### ISSUE TITLE
`SinglePackageProvider` events not working as expected

### ISSUE THREAD

**Issue by jaydensmith** [verbb/postie#91]
Hey @engram-design

It looks like providers using `SinglePackageProvider` haven't been updated to use your new `EVENT_BEFORE_FETCH_RATES` event format, so following the docs doesn't work as expected. Seems `$packedBoxes` is being serialized here:
https://github.com/verbb/postie/blob/a1df8bd874395d56415037a8cda434629a0e7bb7/src/base/SinglePackageProvider.php#L33

**Comment by engram-design** [comment]
Fixed this for the next version. To get the fix early, change your `verbb/postie` requirement in `composer.json` to:

```
"require": {
  "verbb/postie": "dev-craft-3 as 2.4.17",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.4.18](https://github.com/verbb/postie/releases/tag/2.4.18)
